app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -6,15 +6,12 @@ module VerifyProfileConcern
   def url_for_pending_profile_reason
     return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
     return idv_in_person_ready_to_verify_url if current_user.in_person_pending_profile?
-    # We don't want to hit idv_please_call_url in cases where the user
-    # has fraud review pending and not passed at the post office
-    return idv_welcome_url if user_failed_ipp_with_fraud_review_pending?
     return idv_please_call_url if current_user.fraud_review_pending?
     idv_not_verified_url if current_user.fraud_rejection?
   end
 
   def user_has_pending_profile?
-    pending_profile_policy.user_has_pending_profile?
+    pending_profile_policy.user_has_pending_profile? && !user_failed_ipp_with_fraud_review_pending?
```

Comment:
Should this `!user_failed_ipp_with_fraud_review_pending` be part of `PendingProfilePolicy`?

Reply:
I had some back and forth on this. It is a little tough to track exactly what it does. It appears that if a user failed IPP with fraud review pending then their `pending_profile` does not count. That implementation is, perhaps, questionable but I think that is a bigger change than this.

Reply:
It feels like if we know they failed IPP we should cancel the profile